src/Delegates/handlesIndex.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace MeiliSearch\Delegates;
+
+use MeiliSearch\Endpoints\Indexes;
+
+trait handlesIndex
+{
+    public function getAllIndexes()
```

Comment:
> Why is this there no return value type on this method? I get why deleteIndex does not have a return type because MeiliSearch does not return anything, but should this one not be array too?

Declaring the returned type in PHP is not mandatory, but it's a good practice since it has been introduced if I'm not wrong 🙂

I don't have any answer for the second point, I let @ppshobi doing it 😇